### PR TITLE
Add tutorial for running Python scripts

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: source/programming/python.md [hunk 1]
```diff
@@ -138,6 +135,34 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
 ```python
 >>> quit()
 ```
+## Python 脚本
+
+在 Python 解释器的交互模式下，虽然可以执行 Python 代码，但写长代码非常不方便，
+其代码编辑功能很弱，也不具备代码补全功能。更重要的是，当退出交互模式后，
```
Comment (core-man) [inline source/programming/python.md:141]:
```suggestion
其代码编辑功能很弱，也不具备代码补全功能。更重要的是，退出交互模式后，
```

File: source/programming/python.md [hunk 2]
```diff
@@ -138,6 +135,34 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
+之前写的 Python 代码都不会被保存，下次想要执行相同的代码时只能重写。
```
Comment (core-man) [inline source/programming/python.md:142]:
```suggestion
之前写的 Python 代码都不会被保存，下次想要执行相同代码时只能重写。
```

File: source/programming/python.md [hunk 3]
```diff
@@ -138,6 +135,34 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
+因而，通常都不会在 Python 解释器的交互模式下写代码，而是将 Python 代码写到一个
+Python 脚本中保存起来。
+:::{tip}
+Python 解释器交互模式可以当做一个快捷的计算器来使用！
+:::
+
+Python 脚本其实就是一个包含了一系列 Python 脚本的纯文本文件，后缀通常是 `.py`，
```
Comment (core-man) [inline source/programming/python.md:149]:
```suggestion
Python 脚本其实就是一个包含了一系列 Python 指令的文本文件，后缀通常是 `.py`，
```

File: source/programming/python.md [hunk 4]
```diff
@@ -138,6 +135,34 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
+
+下面以一个简单的 Python 脚本作为示例。启动文本编辑器，新建一个文件，将以下
+Python 代码写到文件中：
+```python
+x = 5
+if x > 0:
+    print("Hello world!")
+```
+然后将文件保存为后缀 `.py` 的文件（比如 {file}`first-script.py`），即得到了
+一个可执行的 Python 脚本。打开终端，切换到 Python 脚本所在的目录，然后执行如下命令：
```
Comment (core-man) [inline source/programming/python.md:161]:
```suggestion
一个可执行的 Python 脚本。打开终端，切换到 Python 脚本所在的目录，执行如下命令来运行脚本：
```

File: source/programming/python.md [hunk 5]
```diff
@@ -138,6 +135,34 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
+```
+$ python first-script.py
+```
+Python 脚本会被执行，并在终端输出 `Hello world!`。
```
Comment (core-man) [inline source/programming/python.md:165]:
```suggestion
$ python first-script.py
Hello world!
```

File: source/programming/python.md [hunk 6]
```diff
@@ -138,6 +135,33 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
 ```python
 >>> quit()
 ```
+## Python 脚本
+
+在 Python 解释器的交互模式下，虽然可以执行 Python 代码，但写长代码非常不方便，
```
Comment (core-man) [inline source/programming/python.md:140]:
```suggestion
虽然在 Python 解释器的交互模式下可以执行 Python 代码，但写长代码非常不方便，
```

File: source/programming/python.md [hunk 7]
```diff
@@ -138,6 +135,33 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
+其代码编辑功能很弱，也不具备代码补全功能。更重要的是，退出交互模式后，
+之前写的 Python 代码都不会被保存，下次想要执行相同代码时只能重写。
```
Comment (core-man) [inline source/programming/python.md:142]:
```suggestion
之前写的 Python 代码不会保存，下次想要执行相同代码时只能重写。
```

File: source/programming/python.md [hunk 8]
```diff
@@ -138,6 +135,33 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
+因而，通常都不会在 Python 解释器的交互模式下写代码，而是将 Python 代码写到一个
+Python 脚本中保存起来。
+:::{tip}
+Python 解释器交互模式可以当做一个快捷的计算器来使用！
+:::
```
Comment (core-man) [inline source/programming/python.md:147]: I think the original location is better for this tip.

File: source/programming/python.md [hunk 9]
```diff
@@ -138,6 +135,33 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
 ```python
 >>> quit()
 ```
+## Python 脚本
+
+在 Python 解释器的交互模式下，虽然可以执行 Python 代码，但写长代码非常不方便，
+其代码编辑功能很弱，也不具备代码补全功能。更重要的是，退出交互模式后，
+之前写的 Python 代码都不会被保存，下次想要执行相同代码时只能重写。
+因而，通常都不会在 Python 解释器的交互模式下写代码，而是将 Python 代码写到一个
+Python 脚本中保存起来。
```
Comment (core-man) [inline source/programming/python.md:144]:
```suggestion
因而，通常都不会在 Python 解释器的交互模式下写代码，而是将 Python 代码写到 Python 脚本中。
```

File: source/programming/python.md [hunk 10]
```diff
@@ -138,6 +135,33 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
+脚本的文件名）。
+
+下面以一个简单的 Python 脚本作为示例。启动文本编辑器，新建一个文件，将以下
+Python 代码写到文件中：
+```python
+x = 5
+if x > 0:
+    print("Hello world!")
+```
+然后将文件保存为后缀 `.py` 的文件（比如 {file}`first-script.py`），即得到了
```
Comment (core-man) [inline source/programming/python.md:160]:
```suggestion
将文件保存为后缀 `.py` 的文件（比如 {file}`first-script.py`），即得到了
```

File: source/programming/python.md [hunk 11]
```diff
@@ -138,6 +135,33 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
+一个可执行的 Python 脚本。打开终端，切换到 Python 脚本所在的目录，执行如下命令来运行脚本：
+```
+$ python first-script.py
+Hello world!
```
Comment (core-man) [inline source/programming/python.md:164]:
```suggestion
Hello world!
```
```

**Review by core-man — APPROVED** [review]

File: source/programming/python.md [hunk 12]
```diff
@@ -138,6 +135,32 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
...
 ```
+## Python 脚本
+
+虽然在 Python 解释器的交互模式下可以执行 Python 代码，但写长代码非常不方便，
+其代码编辑功能很弱，也不具备代码补全功能。更重要的是，退出交互模式后，
+之前写的 Python 代码不会保存，下次想要执行相同代码时只能重写。
+因而，通常都不会在 Python 解释器的交互模式下写代码，而是将 Python 代码写到 Python 脚本中。
+:::{tip}
+Python 解释器交互模式可以当做一个快捷的计算器来使用！
+:::
```
Comment (seisman) [inline source/programming/python.md:146]:
```suggestion
```

File: source/programming/python.md [hunk 13]
```diff
@@ -109,9 +109,6 @@ Type "help", "copyright", "credits" or "license" for more information.
 3
 >>>
 ```
```
Comment (seisman) [inline source/programming/python.md:111]:
```suggestion
:::{tip}
Python 解释器交互模式可以当做一个快捷的计算器来使用！
:::
```

File: source/programming/python.md [hunk 14]
```diff
@@ -138,6 +138,30 @@ C 语言使用大括号 `{ }` 划分代码块，而 Python 中使用缩进划分
 ```python
 >>> quit()
 ```
```
Comment (seisman) [inline source/programming/python.md:140]:
~~~suggestion
```

~~~

**Pull request merged**